recipes/ptflops/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "ptflops" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/95/3d/d2af80d6b90d74f1b73e688c287434912fc4416517c6f7505bb6d2ca77d1/ptflops-0.7.3.tar.gz
```

Comment:
Sorry I missed this in my original review, but could you use our canonical `url` definition:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L16

With it hardcoded this way, I'm not sure if the bots that automatically submit PRs when the package is updated upstream will be able to work.